_sass/_settings--fonts.scss

```diff
@@ -1,8 +1,4 @@
 /**
- * Check Bootstrap 3 Documentation for further
- * explanation of some vasriables used below
- * http://getbootstrap.com/customize/#less-variables
```

Comment:
we can change the wording to be Bootstrap 4 but I think the link `http://getbootstrap.com/customize/#less-variables` is still valid in order to check what are the variables supported by bootstrap